app/scripts/controllers/transactions/index.js

```diff
@@ -1474,24 +1479,12 @@ export default class TransactionController extends EventEmitter {
     const signedEthTx = await this.signEthTx(
       unsignedEthTx,
       fromAddress,
-      ///: BEGIN:ONLY_INCLUDE_IN(build-mmi)
-      txMeta.custodyStatus ? txMeta : undefined,
-      ///: END:ONLY_INCLUDE_IN
+      this._getAdditionalSignArguments?.(txMeta),
```

Comment:
Assuming this returns an array, would we not want to use the spread operator to include them all as additional arguments?